mysql-test/t/percona_heap_var.test

```diff
@@ -4,57 +4,56 @@
 --source include/have_innodb.inc
 
 --disable_warnings
-drop table if exists t1;
+DROP TABLE IF EXISTS t1;
```

Comment:
I didn't mean to suggest fixing existing testcase capitalization, but if you have done it, let's keep it